docs/6-release-management/6.2.4-npm.md

```diff
@@ -33,31 +33,28 @@ Research the following commands as well as the `-g` option.
 
 ## SemVer
 
-Watch [Semantic versioning and npm](https://www.youtube.com/watch?v=kK4Meix58R4) on YouTube.
+?> Watch [Semantic versioning and npm](https://www.youtube.com/watch?v=kK4Meix58R4) on YouTube.
 
-## Practice
+## Example Express JS Application
 
-Ensure you have npm, NodeJS, and Java installed your computer. Download Java [here](https://java.com/en/download/) if necessary. (or use Homebrew!)
+1. Follow the Instructions to setup a simple Express JS Application [Express JS Documentation](https://expressjs.com/).
 
-Check your current version of Node with `node -v`.
+2. Make some test for the application by utilizing eslint for the testing [Eslint Documentation](https://eslint.org/).
 
-If you need to install Node, it's recommended to use [Node Version Manager](https://github.com/creationix/nvm/blob/master/README.md).
+3. Make the following scripts in the package.json file
 
-Install Gulp globally with `sudo npm install -g gulp`.
-
-Clone [Dromedary](https://github.com/liatrio/dromedary.git).
-
-```bash
-git clone https://github.com/liatrio/dromedary.git
-cd dromedary
-npm install
+```json
+"scripts": {
+   "start": "node app.js",
+   "test": "eslint your-file-or-folder"
+}
```

Comment:
`eslint` is not a test tool it is a linting tool. Can you just swap in [`jest`](https://jestjs.io/docs/getting-started), prompt them to make a simple test case and update the test script in `package.json`

Reply:
sure let me look into that